var/spack/repos/builtin/packages/r-deriv/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
```

Comment:
```suggestion
# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```

Reply:
Oh, good catch. I have not really been looking for that, but will in the future.

This prompted me to check the current packages that have the wrong copyright. The count is 36.

Reply:
There are 36?!  Ouch.